index.html

```diff
@@ -262,10 +262,7 @@ <h1 class="hidden-xs">NewPipe</h1>
                                 <div class="container">
                                     <div class="details-flex">
                                         <div class="details-image">
-                                            <figure>
-                                                <img src="img/Privacy_written_in_tiles.jpg"/>
-                                                <figcaption>Photo credit: <a href="www.homewatersoftenerreviews.com">Home Water Softener Reviews</a>. License: <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></figcaption>
-                                            </figure>
+                                            <img src="img/Privacy_written_in_tiles.jpg"/>
```

Comment:
> Though, the license allows you to modify the picture. The page contains a recommendation how to credit properly, so I'd suggest you to follow that suggestion, but edit it right into the picture. That is the safest way to ensure compliance.

I added `Photo credit: Home Water Softener Reviews. Photo by Owen More. License: CC BY 2.0` into the photo. That's why I removed the HTML code.